### ISSUE TITLE
"aws-sso-util configure profiles" fails with a ValueError when given a GovCloud region.

### ISSUE THREAD

**Issue by ahublersos** [benkehoe/aws-sso-util#55]
When running `aws-sso-util configure populate --region us-gov-west-1 --dry-run` you get a ValueError. It seems the  [get_short_region()](https://github.com/benkehoe/aws-sso-util/blob/db1832705356eb5a40769b4b8bd04b9776b0e176/cli/src/aws_sso_util/populate_profiles.py#L52) function does not like the format of govcloud regions. The error still appears if `--region-style` is set to `long`.
```
❯ aws-sso-util configure populate --region us-gov-west-1 --dry-run
Logging in to https://start.us-gov-home.awsapps.com/directory/SSOIdentityStoreID
Gathering accounts and roles
Traceback (most recent call last):
  File "/Users/ahubler/.local/bin/aws-sso-util", line 8, in <module>
    sys.exit(cli())
  File "/Users/ahubler/.local/pipx/venvs/aws-sso-util/lib/python3.10/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/ahubler/.local/pipx/venvs/aws-sso-util/lib/python3.10/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/ahubler/.local/pipx/venvs/aws-sso-util/lib/python3.10/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/ahubler/.local/pipx/venvs/aws-sso-util/lib/python3.10/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/ahubler/.local/pipx/venvs/aws-sso-util/lib/python3.10/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/ahubler/.local/pipx/venvs/aws-sso-util/lib/python3.10/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/ahubler/.local/pipx/venvs/aws-sso-util/lib/python3.10/site-packages/aws_sso_util/populate_profiles.py", line 344, in populate_profiles
    profile_name = profile_name_formatter(i, num_regions,
  File "/Users/ahubler/.local/pipx/venvs/aws-sso-util/lib/python3.10/site-packages/aws_sso_util/populate_profiles.py", line 97, in formatter
    kwargs = proc_kwargs(kwargs)
  File "/Users/ahubler/.local/pipx/venvs/aws-sso-util/lib/python3.10/site-packages/aws_sso_util/populate_profiles.py", line 92, in proc_kwargs
    kwargs["short_region"] = get_short_region(kwargs["region"])
  File "/Users/ahubler/.local/pipx/venvs/aws-sso-util/lib/python3.10/site-packages/aws_sso_util/populate_profiles.py", line 53, in get_short_region
    area, direction, num = region.split("-")
ValueError: too many values to unpack (expected 3)
```

One possible solution would be to limit the number of splits in `get_short_region()` to 2 and add some special handling if the value of `direction` is `gov`.

**Comment by nateprewitt** [comment]
While this may not be future proof for new partitions, there are a couple more robust options like:

```python
*area, direction, num = region.split("-")
...
resolved_area = "-".join(area)  # or any other resolution logic for non-default partitions
return "".join([resolved_area, dir_abbr.get(direction, direction), num])
```

or more simply:

```python
area, direction, num = region.rsplit("-", 2)
```

**Comment by benkehoe** [comment]
Fixed in [v4.28](https://github.com/benkehoe/aws-sso-util/blob/master/CHANGELOG.md#cli-v428).
